chromium_src/chrome/browser/chrome_content_browser_client.cc

```diff
@@ -15,8 +17,23 @@
 #define ChromeBrowserMainExtraPartsViewsLinux BraveBrowserMainExtraPartsViewsLinux
 #endif
 
+#define HandleNewTabURLRewrite HandleNewTabURLRewrite_ChromiumImpl
```

Comment:
Without overriding search:: HandleNewTabURLRewrite the newtab page would be handled by chrome. So we replace those handlers with ones that do nothing here.